diff --git a/src/schema.rs b/src/schema.rs
index efd2dac..75e3246 100644
--- a/src/schema.rs
+++ b/src/schema.rs
@@ -19,7 +19,7 @@ diesel::table! {
         yaml_validation -> Bool,
         allow_unsupported -> Bool,
         yaml_limit_per_user -> Nullable<Int4>,
-        yaml_limit_bypass_list -> Array<Nullable<Int8>>,
+        yaml_limit_bypass_list -> Array<Int8>,
         manifest -> Jsonb,
     }
 }
